cmd/kops/update_cluster.go

```diff
@@ -65,6 +65,7 @@ type UpdateClusterOptions struct {
 	SSHPublicKey    string
 	MaxTaskDuration time.Duration
 	CreateKubecfg   bool
+	NodeUpSource    string
```

Comment:
It looks like this isn't used, maybe accidentally included? Also would this override KOPS_BASE_URL?